artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/StompFrame.java

```diff
@@ -103,7 +103,7 @@ public ActiveMQBuffer toActiveMQBuffer() throws Exception {
             return buffer;
          }
 
-         StringBuilder head = new StringBuilder();
+         StringBuilder head = new StringBuilder(512);
```

Comment:
only question I have is how bad was it? do you know.. or you just found this through code reivew?